.github/CONTRIBUTOR_AGREEMENT.md

```diff
@@ -98,9 +98,9 @@ mark both statements:
 
 | Field                          | Entry                |
 |------------------------------- | -------------------- |
-| Name                           |                      |
+| Name                           | Loghi Perinpanayagam                     |
 | Company name (if applicable)   |                      |
-| Title or role (if applicable)  |                      |
-| Date                           |                      |
-| GitHub username                |                      |
+| Title or role (if applicable)  |   Student                   |
+| Date                           |   13 Jan, 2019                    |
+| GitHub username                |   loghijiaha                  |
 | Website (optional)             |                      |
```

Comment:
One small change request: It looks like you accidentally filled in the contributor agreement *template*. Could you copy it over to `.github/contributors` and change the filename to your username (e.g. `.github/contributors/Loghijiaha.md`)?

Reply:
Yes.
Thank you

Reply:
Perfect, thanks! 👍 Could you also reset the changes to `.github/CONTRIBUTOR_AGREEMENT.md`?